Makefile

```diff
@@ -9,3 +14,26 @@ clean:
 	rm -f .coverage .coverage.*
 	find . -name '*.py[co]' -or -name '*.so*' -or -name '*.dylib' -delete
 	find . -depth -name __pycache__ -exec rm -rf {} \;
+
+# LCOV report
```

Comment:
This will send anyone unfamiliar `LCOV` to Google. Can you note what this is? e.g. `# LCOV report (measuring test coverage for C code)`